server/reddit.js

```diff
@@ -31,33 +45,51 @@ redditApi.interceptors.response.use((response) => {
 
 const userAgent = 'node:com.vexillologycontests:v0.1.0';
 
-const snoowrap = new Snoowrap({
-  userAgent,
-  clientId: REDDIT_CLIENT_ID,
-  clientSecret: REDDIT_CLIENT_SECRET,
-  username: 'heshammourad',
-  password: REDDIT_PASSWORD,
-});
+let snoowrap;
+
+if (!IS_DEV) {
+  snoowrap = new Snoowrap({
+    userAgent,
+    clientId: REDDIT_CLIENT_ID,
+    clientSecret: REDDIT_CLIENT_SECRET,
+    username: REDDIT_USERNAME,
+    password: REDDIT_PASSWORD,
+  });
+}
 
+/**
+ * @param {object} [auth] user auth
+ * @returns auth ? user-populated snoowrap : default snoowrap
+ */
 const getSnoowrap = (auth = {}) => {
-  const accessToken = auth.accessToken || auth.accesstoken || auth.access_token;
-  const refreshToken = auth.refreshToken || auth.refreshtoken || auth.refresh_token;
-  if (!accessToken || !refreshToken) {
-    logger.debug('No auth tokens provided, returning default snoowrap');
-    return snoowrap;
+  const accessToken = auth.accessToken || auth.accesstoken || auth.access_token
+    || WEB_APP_ACCESS_TOKEN;
+  const refreshToken = auth.refreshToken || auth.refreshtoken || auth.refresh_token
+    || WEB_APP_REFRESH_TOKEN;
+
+  if (IS_DEV || (accessToken && refreshToken)) {
+    return new Snoowrap({
+      userAgent,
+      clientId: WEB_APP_CLIENT_ID,
+      clientSecret: WEB_APP_CLIENT_SECRET,
+      refreshToken,
+      accessToken,
+    });
   }
 
-  return new Snoowrap({
-    userAgent,
-    clientId: WEB_APP_CLIENT_ID,
-    clientSecret: WEB_APP_CLIENT_SECRET,
-    refreshToken,
-    accessToken,
-  });
+  logger.debug('No auth tokens provided, returning default snoowrap');
+  return snoowrap;
```

Comment:
Seems like there's no longer a way to use the default snoowrap in the DEV environment? We do need to be able to run an unauthenticated access to ensure that users who aren't logged in are able to access the site and see data not specific to their user (though admittedly this would only be for older pages). Instead of looking up `IS_DEV`, why not:
```
  const accessToken = auth.accessToken || auth.accesstoken || auth.access_token
    || WEB_APP_ACCESS_TOKEN;
  const refreshToken = auth.refreshToken || auth.refreshtoken || auth.refresh_token
    || WEB_APP_REFRESH_TOKEN;

  if (!accessToken || !refreshToken) {
    logger.debug('No auth tokens provided, returning default snoowrap');
    return new Snoowrap({
      userAgent,
      clientId: REDDIT_CLIENT_ID,
      clientSecret: REDDIT_CLIENT_SECRET,
      username: REDDIT_USERNAME,
      password: REDDIT_PASSWORD,
    });
  }

  return new Snoowrap({
    userAgent,
    clientId: WEB_APP_CLIENT_ID,
    clientSecret: WEB_APP_CLIENT_SECRET,
    refreshToken,
    accessToken,
  });
```

Reply:
You mentioned on Discord that:

> REDDIT_CLIENT_ID is from a personal use script app I created on Reddit, it's how I used to read data from Reddit but it shouldn't be used anymore

so I never got the unauthenticated snoowrap to work. I think this is a good candidate for the `dotenv`, a variable called `USER_UNAUTHENTICATED`. This way you can set up both the REDDIT_ and WEB_APP_ env variables, but just toggle our `dotenv` to swap between them. 

Lmk if that idea works for you, and we can chat on Discord about getting the default snoowrap running.